uk.co.compucorp.civicrm.hrleaveandabsences/xml/option_groups/leave_and_absences_work_pattern_change_reason_install.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<CustomData>
+
+  <OptionGroups>
+    <OptionGroup>
+      <name>hrleaveandabsences_work_pattern_change_reason</name>
+      <title>Leave and Absence Work Pattern Change Reason</title>
```

Comment:
The name can be only "Work Pattern Change Reason". We don't have this "Leave and Absence" prefix in any of the other option groups of this extension

Reply:
This will be changed.